catalog/input.yaml

```
@@ -18,3 +18,4 @@ feedstocks:
   - https://github.com/leap-stc/WOA-1-degree-monthly-pangeo-forge-osn-feedstock/blob/main/feedstock/catalog.yaml
   - https://github.com/leap-stc/LLC4320_12h_3D-regions_feedstock/blob/main/feedstock/catalog.yaml
   - https://github.com/leap-stc/eNATL_feedstock/blob/main/feedstock/catalog.yaml
+  - https://github.com/leap-stc/chirps_feedstock/blob/main/feedstock/meta.yaml
```

this needs to point to the `catalog.yaml` file instead

```suggestion
  - https://github.com/leap-stc/chirps_feedstock/blob/main/feedstock/catalog.yaml
```